src/pages/ReimbursementAccount/EnableBankAccount/EnableBankAccount.tsx

```diff
@@ -110,30 +100,25 @@ function EnableStep(props) {
                             />
                         )}
                         <MenuItem
-                            title={props.translate('workspace.bankAccount.disconnectBankAccount')}
+                            title={translate('workspace.bankAccount.disconnectBankAccount')}
                             icon={Expensicons.Close}
                             onPress={BankAccounts.requestResetFreePlanBankAccount}
                             wrapperStyle={[styles.cardMenuItem, styles.mv3]}
-                            disabled={Boolean(pendingAction) || !_.isEmpty(errors)}
+                            disabled={Boolean(pendingAction) || !isEmptyObject(errors)}
```

Comment:
```suggestion
                            disabled={!!pendingAction || !isEmptyObject(errors)}
```

Reply:
I actually disagree with this suggestion, in my opinion `Boolean` is less prone to accidental change and conveys the intention better